GitUI/Editor/FileViewer.Designer.cs

```diff
@@ -218,6 +220,14 @@ private void InitializeComponent()
             showSyntaxHighlightingToolStripMenuItem.Text = "Show synta&x highlighting";
             showSyntaxHighlightingToolStripMenuItem.Click += ShowSyntaxHighlighting_Click;
             // 
+            // showGitWordColoringToolStripMenuItem
+            // 
+            showGitWordColoringToolStripMenuItem.Image = Properties.Images.Colors;
+            showGitWordColoringToolStripMenuItem.Name = "showGitWordColoringToolStripMenuItem";
+            showGitWordColoringToolStripMenuItem.Size = new Size(243, 22);
+            showGitWordColoringToolStripMenuItem.Text = "Show Git word coloring";
```

Comment:
To confirm - are we consistent in using "Git" across the app? (vs "git").

Reply:
Not completely, but more Git I believe. This is how Git documenttion describes the tool, but git(.exe) for the executable.